addons/payment_adyen/static/src/js/payment_form.js

```diff
@@ -173,6 +173,35 @@ odoo.define('payment_adyen.payment_form', require => {
                             showPaymentMethods: true,
                             showPayButton: false,
                             setStatusAutomatically: true,
+                            onSelect: (component) => {
+                                if (component.props.name === "PayPal") {
+                                     if (!this.paypalForm){
+                                         // create div element to render PayPal component
+                                         const el = document.createElement("div");
+                                         document.getElementById(
```

Comment:
This will work fine in 15.0, but not in 16.2(?), because Subscriptions displays two payment forms on the same page. Find the inline form (by id if you wish) with a relative `querySelector` instead.

Reply:
As discussed, let's keep it as-is in 15.0, but remember to update in 16.2, and then again in 16.3.